### PR TITLE
result page added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
The result page UI is added in this PR.
please review and give feedback.
Screenshot (123)

### CODE REVIEW

**Review by anshikaa12 — APPROVED** [review]
clean code ! good to merge

File: src/components/resultCard/resultCard.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ .answer-container {
+     display: flex;
+     flex-wrap: wrap;
+     justify-content: space-evenly;
+     row-gap: 1rem;
+     column-gap: 1rem;
+ }
+ 
+ .ans-option {
+
```
Comment (Kothalkarpriya) [inline src/components/resultCard/resultCard.css:25]: remove this space

**Review by Kothalkarpriya — APPROVED** [review]
The code is clean and understandable. You are good to merge the changes

File: src/components/resultCard/resultCard.css (hunk 1 above)
Comment (ShreyashMahajan) [reply src/components/resultCard/resultCard.css:25]: thank you

**Pull request merged**